diff --git a/cobbler/settings.py b/cobbler/settings.py
index b97b11f1..dd14b3a7 100644
--- a/cobbler/settings.py
+++ b/cobbler/settings.py
@@ -131,7 +131,7 @@ DEFAULTS = {
     "sign_puppet_certs_automatically": [0, "bool"],
     "signature_path": ["/var/lib/cobbler/distro_signatures.json", "str"],
     "signature_url": ["https://cobbler.github.io/signatures/3.0.x/latest.json", "str"],
-    "tftpboot_location": ["/srv/tftpboot", "str"],
+    "tftpboot_location": ["/var/lib/tftpboot", "str"],
     "virt_auto_boot": [0, "bool"],
     "webdir": ["/var/www/cobbler", "str"],
     "webdir_whitelist": [".link_cache", "misc", "distro_mirror", "images", "links", "localmirror", "pub", "rendered", "repo_mirror", "repo_profile", "repo_system", "svc", "web", "webui"],
diff --git a/config/cobbler/settings b/config/cobbler/settings
index 169910c8..4fbd4ad1 100644
--- a/config/cobbler/settings
+++ b/config/cobbler/settings
@@ -246,8 +246,8 @@ manage_tftpd: 1
 
 # This variable contains the location of the tftpboot directory. If this directory is not present cobbler does not
 # start.
-# Default: /srv/tftpboot
-tftpboot_location: "/srv/tftpboot"
+# Default: /var/lib/tftpboot
+tftpboot_location: "/var/lib/tftpboot"
 
 # set to 1 to enable Cobbler's RSYNC management features.
 manage_rsync: 0
